crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -2378,6 +2378,35 @@ impl OlmMachine {
         DehydratedDevices { inner: self.to_owned() }
     }
 
+    /// Get the cached dehydrated device pickle key if any.
+    ///
+    /// None if the key was not previously cached (via
+    /// [`Self::save_dehydrated_device_pickle_key`]).
```

Comment:
Oh, another thing. Let's not use `Self::` here, let's expand to the proper struct.